test/test_mxnet.py

```diff
@@ -634,10 +634,6 @@ def test_horovod_alltoall(self):
         rank = hvd.rank()
         size = hvd.size()
 
-        # This test does not apply if using gloo controller
-        if hvd.gloo_enabled():
```

Comment:
There are several other alltoall tests below this that still contain this skip for gloo. Please remove from the remaning alltoall tests.